packages/core/core-flows/src/order/workflows/create-orders.ts

```diff
@@ -39,8 +38,8 @@ export const createOrdersWorkflow = createWorkflow(
       findOrCreateCustomerStep({
         customerId: input.customer_id,
         email: input.email,
-      }),
-      validateVariantsExistStep({ variantIds })
+      })
+      // validateVariantsExistStep({ variantIds })
```

Comment:
should this be removed?

Reply:
I will refactor the order workflows to match all the changes done in carts. If you don't mind I will keep this commented out for now.

The RemoteQuery will validate if the ids exist or not, but it should be possible to customize the message thrown by the RemoteQuery.

Reply:
Sure, let's keep it in